src/constants/index.ts

```diff
@@ -153,4 +153,5 @@ export const GANACHE_LOCAL_RPC_URL = 'http://localhost:8545';
 
 export const isDev = process.env.NODE_ENV === 'development';
 
+export const ACTIVE_INSTALLED_LIMIT = 39;
```

Comment:
I don't think that a correct name maybe sth like `MAX_INSTALLED_NUMBER`

Reply:
Changed